recipes/espsim/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+# NOTE(hadim): https://github.com/conda-forge/libint-feedstock/issues/21
+channel_sources:
+  - conda-forge/label/libint_dev,conda-forge
```

Comment:
Currently blocked by https://github.com/conda-forge/libint-feedstock/issues/21 and https://github.com/evaleev/libint/pull/259